setup.cfg

```diff
@@ -0,0 +1,36 @@
+# Note: setup.cfg is normally a config file for Python package publishing. In
+# our case it's just a shared config file for linting and formatting tools
+# because most of them support it automatically.
+
+[flake8]
+# Try to stick to 79, but sometimes being religious *hurts* readability.
+max-line-length = 100
```

Comment:
Could try a bit lower. It'd be nice if the tools had a notion of "soft limit" and "hard limit" with warning vs. error.